Tests/FluentAssertions.Specs/Equivalency/BasicEquivalencySpecs.cs

```diff
@@ -2083,8 +2083,112 @@ internal class ClassWithoutProperty
         {
         }
 
+#if NET5_0_OR_GREATER
+
+        [Fact]
+        public void Excluding_a_covariant_property_should_work()
+        {
+            var actual = new DerivedWithCovariantOverride(new DerivedWithProperty { DerivedProperty = "a", BaseProperty = "a_base" })
+            {
+                OtherProp = "other"
+            };
+
+            var expectation = new DerivedWithCovariantOverride(new DerivedWithProperty { DerivedProperty = "b", BaseProperty = "b_base" })
+            {
+                OtherProp = "other"
+            };
+
+            actual.Should().BeEquivalentTo(expectation, opts => opts
+                .Excluding(d => d.Property));
+        }
+
+        private class BaseWithProperty
+        {
+            public string BaseProperty { get; set; }
+        }
+
+        private class DerivedWithProperty : BaseWithProperty
+        {
+            public string DerivedProperty { get; set; }
+        }
+
+        private abstract class BaseWithAbstractProperty
+        {
+            public abstract BaseWithProperty Property { get; }
+        }
+
+        private sealed class DerivedWithCovariantOverride : BaseWithAbstractProperty
+        {
+            public override DerivedWithProperty Property { get; }
+
+            public string OtherProp { get; set; }
+
+            public DerivedWithCovariantOverride(DerivedWithProperty prop)
+            {
+                Property = prop;
+            }
+        }
+
+#endif
+
         #endregion
 
+        public interface IInterfaceWithTwoProperties
+        {
+            int Value1 { get; set; }
+
+            int Value2 { get; set; }
+        }
+
+        public class BaseProvidingSamePropertiesAsInterface
+        {
+            public int Value1 { get; set; }
+
+            public int Value2 { get; set; }
+        }
+
+        public class DerivedClassImplementingInterface : BaseProvidingSamePropertiesAsInterface, IInterfaceWithTwoProperties
+        {
+        }
+
+        [Fact]
+        public void Exclude()
```

Comment:
Can we have better test names than `Exclude` and `Include`?

Reply:
Forgot to rename that. Will do.